README.md

```diff
@@ -20,7 +19,30 @@ The `pg` npm package has to be installed. Also – at least node 0.12 or iojs ha
 
 ## Usage
 
-Simple:
```

Comment:
i hope these changes weren't offensive.  i...

- added docs for import and construction,
- added docs for the `options`,
- grouped all the API related things next to each other, then grouped examples together below.